doc/glossary.rst

```diff
@@ -297,17 +297,18 @@
                 Device", so it is important to know which meaning is intended. 
 
 	OSD fsid
-                This is a unique identifier used to identify an OSD. It is
-                found in the OSD path in a file called ``osd_fsid``. The
-                term ``fsid`` is used interchangeably with ``uuid``
+                The OSD fsid is a unique identifier that is used to identify an
+                OSD. It is found in the OSD path in a file called ``osd_fsid``.
```

Comment:
Kinda redundant

Reply:
I want it to be. The Beginner's Guide is going to lead directly to this, and I want there to be a lot of signposting.